NSSServices/Controllers/RegressionRegionsController.cs

```diff
@@ -145,6 +150,11 @@ public async Task<IActionResult> Post(string region, [FromBody]RegressionRegion
                     RegionID = regionEntity.ID,
                     RegressionRegion = entity
                 } };
+                
+                if (entity.Location != null && entity.Location.Geometry.SRID != 102008)
```

Comment:
I'm wondering if this might be better moved to a stored procedure, that handles the conversion at the db level instead of making a few calls back/forth to the db. This will also speed up posts.

Reply:
Ok - I'm not super familiar with stored procedures - where would those go in the code?